packages/components/src/components/resource-link/stories/index.stories.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from "react";
+import { Box } from "../../../primitives/box";
+import {
+  ResourceLink,
+  ResourceLinkCategory,
+  ResourceLinkTitle,
+  ResourceLinkTimeStamp,
+} from "../src";
+
+// eslint-disable-next-line import/no-default-export
+export default {
+  title: "Components/Resource Link",
+  component: ResourceLink,
+};
+
+export const Default: React.FC = () => (
```

Comment:
Just bringing up to start the discussion and you don't need to make the change as part of this PR but what do you think about starting to add more rich documentation to the Forecast components that are visible through Storybook to make it easier to understand when/how to use the components. Something like this: https://github.com/RohitRajendran/components/blob/develop/components/atoms/Button/Button.mdx

Storybook has gotten more advanced with the documentation part so it doesn't need to be in its own mdx file anymore and can just be part of the story definitions and component jsdoc comments.

Reply:
Oh yeah. Good callout. I kind of forgot about all of these new doc features. I'll create a ticket to look into it. Thanks!